diffkemp/simpll/DebugInfo.cpp

```diff
@@ -148,8 +148,8 @@ void DebugInfo::extractAlignmentFromInstructions(GetElementPtrInst *GEP,
                 if (!TypeDISecond)
                     continue;
 
-                int indexSecond =
-                        getTypeMemberIndex(*TypeDISecond, elementName);
+                uint64_t indexSecond = static_cast<uint64_t>(
+                        getTypeMemberIndex(*TypeDISecond, elementName));
```

Comment:
Just use `(uint64_t)` here (and everywhere else). LLVM does the same, so we follow its approach.